lib/pages/dashboard/dashoard_page.dart

```diff
@@ -119,9 +119,11 @@ class _DashboardPageState extends State<DashboardPage> {
                       physics: BouncingScrollPhysics(),
                       itemCount: state.clubs.length,
                       controller: _pageController,
-                      onPageChanged: (pageIndex) => _updateClubTeams(state.clubs[pageIndex].code),
```

Comment:
This method only select the current club even if when selecting the current team, it reloads favorite club's teams